tfconfig/module.go

```diff
@@ -184,6 +184,14 @@ func (r ResourceAttributeReference) MakeRelative(relativePath []string) Relative
 	}
 }
 
+// CopyValues copies attribute values from the other instance while keeping the original metadata (i.e. expression and module references)
+func (r *ResourceAttributeReference) CopyValues(other ResourceAttributeReference) *ResourceAttributeReference {
```

Comment:
IDK, my default thought was its "Copy To" until I read the body & then comment
```suggestion
func (r *ResourceAttributeReference) CopyFrom(other ResourceAttributeReference) *ResourceAttributeReference {
```